### ISSUE TITLE
Need NVDA AddOn for Long tail platinum

### ISSUE THREAD

**Issue by adilsweet** [nvaccess/nvda#7411]
Is there any volunteer or professional who can make an addon for Long tail platinum which is a SEM software. i tried to use it with every curcer but it does not speak everything, some areas are impossible to reach with NVDA and most importantly i cannot click at calculate button, on the other hand, NVDA does not speak anything about keywords and it's unable to move to any other page.

**Comment by adilsweet** [comment]
If someone would like to develop this on demand and ask for a payment then i'm ready to pay for your hardwork but i need this addon to fulfill my needs of my blog

**Comment by josephsl** [comment]
Hi, may I suggest that you contact the app developers about these problems? Once you get a reply from them, please write to us so someone can look for a legal demo of this program and what not. Thanks.

**Comment by leonardder** [comment]
Closing as duplicate of #7403.

**Comment by josephsl** [comment]
Hi, no, the other way around please. Thanks.

**Comment by leonardder** [comment]
[**@josephsl**](https://github.com/josephsl) commented on [17 jul. 2017 18:27 CEST](https://github.com/nvaccess/nvda/issues/7411#issuecomment-315805716 "2017-07-17T16:27:49Z - Replied by Github Reply Comments"):
> Hi, no, the other way around please. Thanks.

Honestly, if it weren't for the duplicate, I'd have closed this due to the fact that this is a request for an add-on. This issue tracker only deals with suggestions or bug reports for NVDA Core.

@adilsweet: I'm sorry, I should have pointed this out in my initial comment. However, even if you'd have requested this and didn't suggest using cracked software, it would have been very unlikely that NV Access would invest resources in a paid app hardly anybody uses.

Please investigate the [connect](https://github.com/nvaccess/nvda-community/wiki/Connect) page in order to find ways to get help. One thing you could do, for example, is send your request to the international users' mailing List. Alternatively, you could address your quest to the [NVDA add-ons discussion forum](https://addons.nvda-project.org/addons/groupsio.nl.html). Again, I'm sorry if I sounded harsh, it wasn't my intention to do so.

**Comment by adilsweet** [comment]
thanks for replies, i have a licence for myself, i can share with you because i have permission to install it with 3 computers, if you are able to make i'll share my licence with you till then you want, i pay 352 dollars yearly.

**Comment by josephsl** [comment]
Hi, personally, I’d still say no because this raises ethical questions. Even if we want to add a feature, if it runs into ethical problems such as improper license transfers, it would have been better to think about alternatives. Thanks.

**Comment by leonardder** [comment]
[**@adilsweet**](https://github.com/adilsweet) commented on [18 jul. 2017 05:25 CEST](https://github.com/nvaccess/nvda/issues/7411#issuecomment-315947598 "2017-07-18T03:25:44Z - Replied by Github Reply Comments"):
> i have a licence for myself

Even though you can install on multiple pc's, you are probably not allowed to share a personal licence.

**Comment by adilsweet** [comment]
means there is no way to get an addon. so what if i purchase a new licence for another computer and if owner of the software allows me to make an addon for NVDA? if it's still impossible then no problem, i don't mind.

**Comment by josephsl** [comment]
Hi, at this stage, I think it’d be best to talk to Long Tail Pro developers directly so folks using various screen readers can benefit from it, particularly if it’s a standalone app and not a website; ideally solutions should come from the vendor. As a customer, if you have genuine license, you are in a position of influence. Please talk to app vendor first regarding accessibility problems, then come back if folks say this is a screen reader problem and is asking for advice from screen reader developers. Thanks.

**Comment by leonardder** [comment]
Agreed with @josephsl. Furthermore, please address further questions to the NVDA mailing list, GitHub is related to NVDA core issues only, not for add-on discussions or assistance regarding making software accessibility.
